.github/workflows/unifiedBuildTestAndInstall.yml

```diff
@@ -95,6 +95,10 @@ on:
         description: "Install steps to run"
         required: false
         type: string
+      package_name_prefix:
+        description: "The prefix for package name"
+        required: true
+        type: string
```

Comment:
Can you include this in the `workflow_dispatch` inputs so that this can be run manually?

Reply:
Sure, done.